build.rs

```diff
@@ -6,28 +6,32 @@ use std::fs;
 use std::path::Path;
 
 #[cfg(windows)]
-fn platform_specific() {
+fn platform_specific() -> std::io::Result<()> {
 	let mut res = winres::WindowsResource::new();
 	res.set_icon("resource_dev/emulsion.ico");
-	res.compile().unwrap();
+	res.compile()?;
```

Comment:
Returning an error instead of panicing doesn't provide any advantage. If we cannot create the `WindowsResource` here, the best course of action is to panic. The compiler will then print the panic message and we can adjust our build script accordingly.